utils.py

```diff
@@ -891,11 +891,22 @@ def check_auth(user, hutch):
 def check_special(ioc, hutch):
     lines = open(SPECIAL_FILE % hutch).readlines()
     lines = [l.strip() for l in lines]
-    for l in lines:
+    for l in lines: #Note: (josh) as long as the ioc name is in the special file entry we let it be cycled by any user 
```

Comment:
So, if you want to upgrade versions *and* allow enabling/disabling, you want two lines in iocmanager.special?

Reply:
you're right seems overkill, lets say if the entry is in the special file period that ioc should be able to enable / disable. And then it can only upgrade if it adheres to the schema outlined in this PR .

So not two lines